### DISCUSSION TITLE
[상세페이지] 변수명 논의

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#72] [Ideas]
<img width="417" alt="스크린샷 2022-06-10 오후 6 14 46" src="https://user-images.githubusercontent.com/72402747/173033327-54c69f72-ce4e-4ee3-966f-64858e15e496.png">

**Comment by Hyevvy** [comment]
제목 title
모집분야 parts 

parts: [{channel: 'front', people: 5, skills:[["html5"], ["next.js"]] }, {channel:'back', people:10, skills:[]}]


진행방식 place

```
  const placeOptions = [
    { id: 1, value: "online", label: "온라인" },
    { id: 2, value: "offline", label: "오프라인" },
    { id: 3, value: "both", label: "온오프라인 병행" },
    { id: 4, value: "notyet", label: "미정" }
  ];
```


모집인원 people

시작예정일 startDate

연락방법-> 이메일로 변경하고 email

예상기간 -> expectedDate

기술스택 -> skills


(프론트만 임시로 작성해보았습니다)

```
const skillOptions= [
{channel: front , options: [  { id: 1, value: "Html5", label: "Html5" },
    { id: 2, value: "TypeScript", label: "TypeScript" },
    { id: 3, value: "React", label: "React" },
    { id: 4, value: "Next.js", label: "Next.js" },
    { id: 5, value: "Vue", label: "Vue" },
    { id: 6, value: "Nuxt.js", label: "Nuxt.js" },
    { id: 7, value: "Svelte", label: "Svelte" },
    { id: 8, value: "Gatsby", label: "Gatsby" },
    { id: 9, value: "Angular", label: "Angular" },
    { id: 10, value: "Sass", label: "Sass" },
    { id: 11, value: "Redux", label: "Redux" },
    { id: 12, value: "Recoil", label: "Recoil" },
    { id: 13, value: "Bootstrap", label: "Bootstrap" },
    { id: 14, value: "Mui", label: "Mui" },
    { id: 15, value: "Tailwind", label: "Tailwind" }] },
{channel: }
]
```
이미지 image | Optional<String>,


====================== 
서버로부터 post 에 대해 받는 값

{
  "likes": Like[],
  "comments": Comment[],
  "_id": String,
  "image": Optional<String>,
  "imagePublicId": Optional<String>,
  "title": String,
  "channel": Channel,
  "author": User,
  "createdAt": String,
  "updatedAt": String
}

**Comment by Hyevvy** [comment]
특정 채널에 글 작성시 변수명
<img width="503" alt="스크린샷 2022-06-10 오후 6 18 24" src="https://user-images.githubusercontent.com/72402747/173034012-d2d0bde5-5c87-4f1a-8c3f-c470ca423c02.png">

**Comment by hwoo3303** [comment]
## Post 추가 데이터
- 프로젝트 소개(introduction)

**Comment by NamgyungKim** [comment]
✅
```js
// type 전부 string 
title:  "{
        'title': '✨샘플',
        'introduction': 'test중 입니다.',
        'email': 'knk@gmail.com',
        'expectedDate': '1개월',
        'place': 'online',
        'startDate' : '2022/06/06',
        'parts': {
                       channel: 'front', 
                        people: '5', 
                        skills: [ ['html5'], ['next.js'] ]
                }
        }"
```

**Reply by Hyevvy** [reply]
startDate

**Reply by Hyevvy** [reply]
parts:{channel: 'front', people: 5, skills:[["html5"], ["next.js"]]}

**Reply by Hyevvy** [reply]
@NamgyungKim 여기용

**Comment by Hyevvy** [comment]
people string 입니당 ㅎㅎ
<img width="347" alt="스크린샷 2022-06-16 오후 5 32 20" src="https://user-images.githubusercontent.com/72402747/174028415-767aa71c-3eb1-4435-aaac-7e093f81ec52.png">

**Comment by Hyevvy** [comment]
<img width="371" alt="스크린샷 2022-06-16 오후 5 42 47" src="https://user-images.githubusercontent.com/72402747/174030589-16572cc5-621c-4062-9f8d-21eea82317f8.png">

**Reply by Hyevvy** [reply]
const placeOptions = [
  { id: 1, value: "online", label: "온라인" },
  { id: 2, value: "offline", label: "오프라인" },
  { id: 3, value: "both", label: "온오프라인 병행" },
  { id: 4, value: "notyet", label: "미정" }
];

**Reply by Hyevvy** [reply]
onst expectedDateOptions = [
  { id: 1, value: "1개월", label: "1개월" },
  { id: 2, value: "2개월", label: "2개월" },
  { id: 3, value: "3개월", label: "3개월" },
  { id: 4, value: "4개월", label: "4개월" },
  { id: 5, value: "5개월", label: "5개월" },
  { id: 6, value: "6개월 이상", label: "6개월 이상" },
  { id: 7, value: "notyet", label: "미정" }
];

**Comment by Hyevvy** [comment]
@NamgyungKim  이런 식으로 서버에 값을 보내게 됩니다! 타입은 string입니당
<img width="328" alt="스크린샷 2022-06-16 오후 6 02 18" src="https://user-images.githubusercontent.com/72402747/174034441-28287893-3518-4aff-a824-b87ddc78af5f.png">
